core/src/test/java/com/google/googlejavaformat/java/MainTest.java

```diff
@@ -238,6 +242,9 @@ public void importRemovalLines() throws Exception {
   // test that errors are reported on the right line when imports are removed
   @Test
   public void importRemoveErrorParseError() throws Exception {
+    Locale backupLocale = Locale.getDefault();
+    Locale.setDefault(Locale.ROOT);
```

Comment:
Do we understand why the diagnostics are being affected by the default locale. As I mentioned earlier, that isn't supposed to happen: https://github.com/google/google-java-format/pull/195/commits/996962604f2b772a14aa9729d9fc8ae7d9dd7df1#r140948765

Reply:
Hmm... Sorry @cushon, but I don't see a comment when I click on that link, so it's not clear to me what you're referring to. :(

Reply:
Do this work?

`https://github.com/google/google-java-format/pull/195/commits/996962604f2b772a14aa9729d9fc8ae7d9dd7df1#r140948765`